### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by k9bbzJavftZaqAqSW3dY** [Mutcholoko/Haptic-Feedback-Linux#1]
Very cool git! I was easily able to compile this with the userpatches loader in the wine-tkg build method. It just worked. I didn't even need the udev rule, maybe they fixed the identification upstream somewhere - I'm using pipewire, linux 6.1-RC, and the newest valve proton. Just wanted to drop by and let you know that this repository made my day :)

**Comment by Mutcholoko** [comment]
I'm glad to hear this! It also made my day when I discovered that it was possible, but in the end we should all praise @ClearlyClaire for finding out about this. I'm curious to see if this will ever get implemented on the default Proton. 

Thanks a lot for your comment!

**Comment by SvdB-nonp** [comment]
Is it possible to do a release for a more recent proton version? I tried your guide but unfortunately it did not work for me. Tested it with Returnal, which your published proton with the fixes works beautifully for, but my own compilated proton refuses to have any haptics working as if the patches did not work...

Currently playing The Last of Us, but it unfortunately requires fixes in Proton Experimental it this moment. Since the steps you outlined did not work for me, I am wondering if you could publish a proton-build for a more recent proton.

**Comment by Mutcholoko** [comment]
> Is it possible to do a release for a more recent proton version?

If I'm not mistaken, Claire is now making builds too on [her repo](https://github.com/ClearlyClaire/wine), but I'm not sure if the latest build includes the Proton Experimental fixes.

>I tried your guide but unfortunately it did not work for me

It seems that Valve changed the folder called "Wine" to "wine", so that might be what was giving you trouble. I'll fix the readme! Thanks for the heads up :)

**Comment by SvdB-nonp** [comment]
> If I'm not mistaken, Claire is now making builds too on [her repo](https://github.com/ClearlyClaire/wine), but I'm not sure if the latest build includes the Proton Experimental fixes.

Thanks a bunch, will try those. I did not notice she had releases when I checked her repo.

And regardless, thanks for the proton build of september until now. Enjoyed Uncharted and Returnal with great pleasure with it.

**Comment by ClearlyClaire** [comment]
Yes, I had to rebuild Proton so I took the opportunity to redistribute it. It's from a week ago or something, I am not sure what fixes you expect.

**Comment by SvdB-nonp** [comment]
With the older release of @Mutcholoko Plague Tale Requiem does not launch for me, but it does with current proton experimental. So no option to have haptics for that game before today.

Fortunately, I can confirm that your release does work splendidly haptics-and-all :)
